docs/clusters/santis.md

```diff
@@ -1,13 +1,17 @@
 [](){#ref-cluster-santis}
 # Santis
 
-Santis is an Alps cluster that provides GPU accelerators and file systems designed to meet the needs of climate and weather models for the [CWp][ref-platform-cwp].
+Santis is an Alps cluster that provides GPU accelerators and file systems designed to meet the needs of climate and weather models for the [CWP][ref-platform-cwp].
 
 ## Cluster specification
 
 ### Compute nodes
 
-Santis consists of around ??? [Grace-Hopper nodes][ref-alps-gh200-node].
+Santis consists of around 600 [Grace-Hopper nodes][ref-alps-gh200-node].
```

Comment:
It still says 1200 nodes below in the table. Should that stay like this as long as gb preparations are ongoing or should it list the "normal" number of nodes?

Reply:
I added a note in the line below this that there has been a temporary increase to 1200.

I will update the table.